src/libraries/System.ComponentModel.TypeConverter/src/System/ComponentModel/Design/DesigntimeLicenseContext.cs

```diff
@@ -77,16 +77,13 @@ public override string GetSavedLicenseKey(Type type, Assembly resourceAssembly)
                     // try everything.
                     foreach (Assembly asm in AppDomain.CurrentDomain.GetAssemblies())
```

Comment:
I am not familiar with this code, so I need to defer to someone else to sign off on the changes to this method.